packages/@tailwindcss-vite/src/index.ts

```diff
@@ -8,6 +8,7 @@ import type { Plugin, ResolvedConfig, Rollup, Update, ViteDevServer } from 'vite
 
 const DEBUG = env.DEBUG
 const SPECIAL_QUERY_RE = /[?&](raw|url)\b/
+const INLINE_STYLE_ID_RE = /[?&]index\=\d+\.css$/
```

Comment:
Interesting, is this always the case? I'm assuming we see the link as `/index.html?index=0.css` where `0` is the index of the `<style>` block on the page?

Reply:
Yep, that's what I saw when i tested it 👍